test/src/org/labkey/test/tests/mobileappstudy/BaseMobileAppStudyTest.java

```diff
@@ -247,7 +247,7 @@ protected String submitSurvey(Survey survey, int expectedStatusCode)
 
     protected void setSurveyMetadataDropDir()
     {
-        ModulePropertyValue val = new ModulePropertyValue("MobileAppStudy", "/", "SurveyMetadataDirectory", TestFileUtils.getLabKeyRoot() + "/server/optionalModules/mobileAppStudy/test/sampledata/SurveyMetadata/");
+        ModulePropertyValue val = new ModulePropertyValue("MobileAppStudy", "/", "SurveyMetadataDirectory", TestFileUtils.getLabKeyRoot() + "/server/modules/mobileAppStudy/test/sampledata/SurveyMetadata/");
```

Comment:
This shoudn't need to know where the module is enlisted:
```suggestion
        ModulePropertyValue val = new ModulePropertyValue("MobileAppStudy", "/", "SurveyMetadataDirectory", TestFileUtils.getSampleData("SurveyMetadata").getAbsolutePath());
```